utils/material_plots_2D.py

```diff
@@ -1,17 +1,44 @@
+"""
+This script must be called with python: 'python material_scan_2D.py --{argument} {value}'.
+The output files are saved in data/{outputDir}/name.suffix.
+If no outputDir is specified, it will be data/plots/name.suffix.
+"""
+
 from __future__ import print_function
+
 import argparse
 import math
-
-import sys, os
+import sys
+from os import fspath
+from os.path import expandvars
+from pathlib import Path
 
 sys.path.append(os.path.expandvars("$FCCSW") + "/Examples/scripts")
 from plotstyle import FCCStyle
 import ROOT
 
 
+def create_histogram(name_and_title: str, argument_name_space: argparse.Namespace) -> ROOT.TH2F:
```

Comment:
```suggestion
def create_histogram(name_and_title: str, argument_name_space: argparse.Namespace) -> ROOT.TH2F:
```
Can we make this take the number of bins as well as the min/max values instead of an `argument_name_space`? That would make this a bit more re-usable and less dependent to work only for these specific arguments.